src/driver/slack.ts

```diff
@@ -248,9 +248,20 @@ export class DeborahDriverSlack extends DeborahDriver
 		}
 		// sendAs関数に処理を引き渡す
 		var m: DeborahMessageSlack = <DeborahMessageSlack> replyTo;
-		this.sendAs(m.getChannelID(), "@"+replyTo.senderName+" "+message, this.bot.settings.profile.name, this.bot.settings.profile["slack-icon"]);
+		
+		//typing indicator
+		var {RTMClient} = require('@slack/client');
+		var rtm = new RTMClient(this.settings.token);
+		rtm.start();
+		rtm.sendTyping(m.getChannelID());
+		setTimeout(() => {
+			//rtmのほうがTypingIndicatorとの時間差が少ないがslack-iconの指定ができない
+			//rtm.sendMessage("@"+replyTo.senderName+" "+message, m.getChannelID());
+			this.sendAs(m.getChannelID(), "@"+replyTo.senderName+" "+message, 
```

Comment:
I think we should use rtm.sendMessage() instead of this.sendAs(). Please find a way to specify slack-icon when using rtm.sendMessage().